templates/community-viz/package.json

```diff
@@ -5,15 +5,14 @@
     "jsFile": "index.js",
     "jsonFile": "index.json",
     "cssFile": "index.css",
-    "manifestFile": "manifest.json",
     "print": "printMessage.js"
   },
   "scripts": {
-    "build:dev": "node scripts/dscc-scripts.js build prod:false",
-    "build:prod": "node scripts/dscc-scripts.js build prod:true",
-    "push:dev": "node scripts/dscc-scripts.js push prod:false",
-    "push:prod": "node scripts/dscc-scripts.js push prod:true",
-    "updateMessage": "node scripts/dscc-scripts.js updateMessage",
+    "build:dev": "node scripts/dscc-scripts.js --script=build --prod=false",
+    "build:prod": "node scripts/dscc-scripts.js --script=build --prod=true",
+    "push:dev": "node scripts/dscc-scripts.js --script=push --prod=false",
+    "push:prod": "node scripts/dscc-scripts.js --script=push --prod=true",
+    "updateMessage": "node scripts/dscc-scripts.js --script=updateMessage",
```

Comment:
Nit: could you make updateMessage be "update_message". Most command line things I see use snake_case instead of camelCase.